services/QuillLMS/app/controllers/api/v1/active_activity_sessions_controller.rb

```diff
@@ -32,6 +32,15 @@ def destroy
     render(plain: 'OK')
   end
 
+  private def working_params
+    permitted_passage_object = params[:active_activity_session][:passage].map do |paragraph| 
+      paragraph.map do |word_object|
+        word_object.permit!
+      end
+    end
+    valid_params.merge({ passage: permitted_passage_object })
```

Comment:
Is this going to save a value to `passage` even for non-Proofreader sessions?

Reply:
Sorry, just added a guard for this - refresh the PR :-)